setup.py

```diff
@@ -10,6 +10,13 @@
 logger = get_logger()
 
 
+if not settings.model.pre_download_during_build:
+    logger.info("Skipping model pre-download as it is disabled in settings.yaml")
+    exit(0)
+else:
+    logger.info("Pre-downloading models... WARNING: You likely don't want to do this!! This will make the image large.")
+
+
```

Comment:
Please, nest the rest of the code under else and drop the exit(0).

Reply:
Will the model be downloaded into the image partition in `/cache`? it should then be attached as external storage then.

Reply:
This is a part of the build process, but the environment variable EVA_AI_MODEL__CACHE_DIR controls where the models are downloaded. Ideally, you should set the "download at build" env var to false in github vars. It's there just for testing.

Reply:
Sure, I'm asking because of the eventual deployment.